services/121-service/test/payment/do-payment-fsp-visa-debit/do-payment-fsp-visa-debit.success.test.ts

```diff
@@ -226,13 +220,13 @@ describe('Do succesful payment with FSP Visa Debit', () => {
       accessToken,
       maxWaitTimeMs: 6_000,
       completeStatusses: Object.values(TransactionStatusEnum),
-      payment: testPaymentNumber,
+      paymentId: testPaymentNumber,
```

Comment:
I think 'testPaymentNumber' should be removed and use id from doPaymentResponse instead